content/docs/next-release/distributed-tracing/index.adoc

```diff
@@ -0,0 +1,115 @@
+
+image::small_form.png[Smal Form Tracing View]
+
+
+The service selector has the list of services of the selected namespaces organized by namespace.
+
+image::service.png[Service Selector Tracing View]
+
+
+For an advanced search of traces we can expand the "Show advanced options"
+
+image::full_form.png[Full form Tracing View]
+
+
+* Tags: Empty by default, find the traces with these values. Example: `http.status_code=200`,error=true`
+* Limit results: 20 by default, number of traces results.
+* Span Configurations (More info about spans link:https://www.jaegertracing.io/docs/1.9/architecture/#span[here, window="_blank"])
+** Min Duration: Min duration for a span. Eg: 1.2s, 100ms, 500us
+** Max Duration: Max duration for a span. Eg: 1.2s, 100ms, 500us
+
+
+== Kiali and Tracing Service have not the same protocol
+
+In this case you'll have a link to open the service in a new tab in your browser in the menu of Kiali and in the view of a specific service.
+
+image::not_same_protocol.png[Not same protocol Kiali and Tracing]
+
+
+== Help to Debug. Troubleshooting
+
+
+=== I can't see the link to Tracing or traces tab in service details.
+
+Check in the about modal if you have the JaegerURL variable with the URL
+
+image::about_menu.png[About menu]
+
+image::about.png[About modal]
+
+
+
+=== I have no Tracing URL in the about modal.
+
+So Kiali didn't find the tracing service. Kiali tries to search for the tracing service in `tracing` or `jaeger-query`. If you have another name for this service you must set the values in the link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[yaml configuration, window="_blank"] for link:https://operatorhub.io/operator/alpha/kiali-operator[Kiali Operator, window="_blank"]
+
+```yaml
+# **Tracing-specific settings:
+#  - Right now we only support Jaeger
+# namespace: The Kubernetes namespace that holds the Tracing service (if empty, assumes the same value as deployment.namespace)
+# service: The Kubernetes service name for tracing. Kiali uses this to connect within the cluster to Jaeger.
+# url: The URL that Kiali uses when integrating with Tracing. This URL must be accessible to clients external to
+#      the cluster in order for the integration to work properly. If empty, an attempt to auto-discover it is made.
+#    ---
+#    tracing:
+#      namespace: ""
+#      service : ""
+#      url: ""
+```
+
+
+=== I can't see the embedded feature, there is a link to a new tab.
+
+Check the about modal like the link:http://localhost:1313/documentation/tracing/#_i_can_t_see_the_link_to_tracing_or_traces_tab_in_service_details[step 3.1] if you have the same protocol link:https://en.wikipedia.org/wiki/HTTPS[HTPPS, window="_blank"] or link:https://en.wikipedia.org/wiki/HTTP[HTTP, window="_blank"].
```

Comment:
use a relative url instead of `http://localhost:1313`.. As this refers to the same page, only the anchor part is needed (`#_i_can..`).